components/liquidity/LiquidityModal.tsx

```diff
@@ -65,7 +65,9 @@ const LiquidityModal = ({
     })
     ?.data.balance.free.toString();
 
-  const baseAsset = parseAssetId(pool?.baseAsset).unrightOr(null);
+  const baseAsset = pool?.baseAsset
```

Comment:
this can be written as `const baseAsset = pool && parseAssetId(pool.baseAsset).unrightOr(undefined);`

`&&` operator returns the value of the first falsy operand encountered when evaluating from left to right, or the value of the last operand if the first is truthy, so baseAsset will be undefined if pool is undefined and if pool is defined it will return the value of `parseAssetId(pool.baseAsset).unrightOr(undefined)`